### ISSUE TITLE
[BUG]Incorrect Scale for Anan-10E only when using DUP+PS+TUN+TONE

### ISSUE THREAD

**Issue by ramdor** [ramdor/Thetis#215]
Reported via email

When selecting DUP and PS while transmitting TUN or TONE, the display is not showing correctly. If the receive channel is offset from the centre, ie. CTUN is on, the display shows a scale at the receive point but displays the transmit signal in the centre of the display. Switching either of DUP or PS corrects the display. 

It only seems to be doing this for the Anan-10E.

**Comment by ramdor** [comment]
I have quickly looked at this and it is non trivial. When PS + DUP is used on a single rx dual ddc radio, PS needs to be using a ddc that is essentially at tx frequency. The issue has been in the software for ages and will take some time to resolve.

**Comment by RdWing** [comment]
I also run into this when using Thetis with an HL2. It's at its worst when using CTUN+DUP+PS-A.

**Comment by ramdor** [comment]
yes it will take a fair bit of work to fix. I may start work on it sometime later in the month.

**Comment by ramdor** [comment]
hermes
anan10
anan10e
anan100
anan100b
